packages/components/src/Banner/Banner.test.tsx

```diff
@@ -92,7 +92,23 @@ it("renders a banner with a primary 'learning' action when the type is 'notice'"
   expect(tree).toMatchSnapshot();
 });
 
-test("it should call the onClick when primaryAction is present", () => {
+it("does not wrap the its children in text if the children are not a simple string", () => {
+  const tree = renderer
+    .create(
+      <Banner
+        type="notice"
+        primaryAction={{
+          label: "smash me",
+        }}
+      >
+        <h3>Bruce</h3>
+      </Banner>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
a snapshot test? how dare you 😛

Reply:
You can probably do a find on `Bruce` and check if the parent is an instance of `HTMLDivElement`. If it gets wrapped in a `<Text />` then that's an `HTMLParagraphElement`

Reply:
I know, I figured your team will take this own and you can clean this garbage 😛 
But in the spirit of giving a shit I will change this!